python/artm/model.py

```diff
@@ -638,8 +640,11 @@ def get_phi(self, topic_names=None, class_ids=None):
         if not self._initialized:
             raise RuntimeError('Model does not exist yet. Use ARTM.initialize()/ARTM.fit_*()')
 
-        phi_info = self.master.get_phi_info(model=self.model_pwt)
-        nd_array = self.master.get_phi_matrix(model=self.model_pwt,
+        if model_name is None:
+            model_name = self.model_pwt
```

Comment:
Don't change model_name, it's an argument passed by reference. Use smth like this

final_model_name = model_name if model_name is not None else self.model_pwt
phi_info = self.master.get_phi_info(model=final_model_name)
nd_array = self.master.get_phi_matrix(model=final_model_name)